mailbox/api/src/main/java/org/apache/james/mailbox/MailboxManager.java

```diff
@@ -397,6 +397,14 @@
 
 
     /**
+     * Reset the Mailbox ACL of the designated mailbox.
+     *
+     * @param mailboxACL New ACL value
+     * @throws UnsupportedRightException
+     */
+    void resetRights(MailboxPath mailboxPath, MailboxACL mailboxACL, MailboxSession session) throws MailboxException;
```

Comment:
what is the difference with REPLACE mode ?

Reply:
The fact that it concerns several 'acl keys' ?

Reply:
I understand. The method name is probably ambiguous : reset should not take a new definition, it should rather put it in the default state.
I thing the problem comes from the fact setRights is not the right name either because it doesn't set the given rights, it applies a ACLCommand.
If you keep setRights to define a full MailboxACL, it starts to make sense.
Then you could rename setRights to applyRightsCommand. WDYT ?

Reply:
The only downside is API stability here.

However, I think we are the first one to use ACL, and that we massively change ACLs APIs in the refactoring commits. So let's do this move.